src/common/utils/get_storage_recipe.py

```diff
@@ -9,11 +9,18 @@
     **{
         "name": "Yaml",
         "type": SIMOS.UI_RECIPE.value,
-        "plugin": "yaml",
+        "plugin": "@development-framework/dm-core-plugins/yaml",
         "category": "view",
     }
 )
-default_form_edit = Recipe(**{"name": "Edit", "type": SIMOS.UI_RECIPE.value, "plugin": "form", "category": "edit"})
+default_form_edit = Recipe(
+    **{
+        "name": "Edit",
+        "type": SIMOS.UI_RECIPE.value,
+        "plugin": "@development-framework/dm-core-plugins/form",
+        "category": "edit",
```

Comment:
Shoud we remove the category?

Reply:
I cannot see that category is used anywhere in the dm-core-packages code.
should it be removed from UiRecipe blueprint in DMSS?